src/System.Security.Cryptography.Cng/src/System.Security.Cryptography.Cng.builds

```diff
@@ -6,28 +6,12 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Security.Cryptography.Cng.csproj">
-      <OSGroup>Windows_NT</OSGroup>
-      <TargetGroup>netstandard1.6</TargetGroup>
```

Comment:
I don't know that we should turn off the netstandard1.6 or netcore50 builds.  If we've made bugfixes to the core part of the library they should be rebuilt, not just carried forward.  The facades are fine to dump, though.